### PR TITLE
Created one print function (and deleted the code duplicated ones), + added index

### PR DESCRIPTION
**Pull request by morbenami1**

### CODE REVIEW

File: src/main/java/UI.java [hunk 1]
```diff
@@ -18,28 +18,28 @@ public class UI {
...
     private static final String PATIENT_MAIN_MENU =
             "This is the Patient Main Menu!" + System.lineSeparator()
-            + "List of commands: "
-            + UI.PATIENT_ADD
-            + UI.PATIENT_VIEW_ALL
-            + UI.PATIENT_RETRIEVE
-            + UI.PATIENT_EDIT
-            + UI.RETURN_TO_MAIN
-            + UI.EXIT_PROGRAM;
+                    + "List of commands: "
```
Comment (kaseykwok) [inline src/main/java/UI.java:21]: May consider removing the extra tabs as 8 spaces should be fine.

File: src/main/java/UI.java [hunk 2]
```diff
@@ -102,10 +102,11 @@ public void printPatientMenuMessage() {
         System.out.println(PATIENT_MAIN_MENU);
     }
 
-    public void printPatientAddedMessage(Patient patient) {
-        System.out.println("Ok! I've added a patient! The patient's details are as follows:");
+    public void printMessageAndObject(String object, String message, int index) {
+        System.out.println(message);
         printLine();
-        System.out.println(patient);
+        System.out.println((index + 1) + ")");
```
Comment (kaseykwok) [inline src/main/java/UI.java:108]: I think index should be printed in `Prescription #1` or `Visit #1` format to align with the other viewing functions, you may check the print messages for viewing functions e.g viewAll

**Review by kaseykwok — CHANGES_REQUESTED** [review]
Please consider aligning the index format

File: src/main/java/UI.java (hunk 1 above)
Comment (morbenami1) [reply src/main/java/UI.java:21]: not my code lol but sure, you can check next time who wrote it...

Comment (kaseykwok) [reply src/main/java/UI.java]: but you can look at the file changed, you can see the extra tabs you commited

**Pull request merged**